.circleci/config.yml

```diff
@@ -1192,26 +1198,33 @@ workflows:
       - run-test-ios-16:
           xcode_version: '14.3.0'
           install_swiftlint: false
+          <<: *release-branches
```

Comment:
Hmm maybe I'm wrong but the `generate-snapshot` and `geenerate_revenuecatui_snapshots` are not run on every commit but on demand, in order to update the backend requests snapshots and UI snapshots respectively.

I think we shouldn't filter these to only run on release jobs in case we want to update the test snapshots in a normal PR? (Do lmk if I missed something)

Reply:
you're absolutely right, that was a pretty terrible oversight from me